### PR TITLE
Change the email address

### PR DESCRIPTION
**Pull request by zairdon20**
In the following milestone:

- Made sure I deploy the website using GitHub Pages.

- Made sure that I  the online version of my portfolio and that the page works properly.

- Made sure I update the README of your repository to include a link to the online version.

### CODE REVIEW

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @zairdon20, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [x] Having looked at your file changed, you seem to have raised a pull request for deployment but submitted the link through the accessibility project. Please check your submission.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Additional description about the project and its features.
 
 [Live Demo Link](https://zairdon20.github.io/JOSPHATS-FIRST/)
 
-
+[Online view link ](https://zairdon20.github.io/JOSPHATS-FIRST/)
```
Comment (rustam-tolipov) [inline README.md:21]: - [x] [OPTIONAL] You need only one link! I think `Live Demo Link` is enough for it! 😊
```suggestion
```

**Review by rustam-tolipov — APPROVED** [review]
## Hi  @zairdon20   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (zairdon20) [reply README.md:21]: Thanks for the advice

**Pull request merged**